src/main/java/no/nav/familie/inntektsmelding/forespørsel/modell/ForespørselRepository.java

```diff
@@ -94,6 +94,19 @@ public void oppdaterSakId(UUID forespørselUUID, String sakId) {
         entityManager.flush();
     }
 
+    public void settForespørselTilUtgått(String sakId) {
```

Comment:
Kanskje vi burde hatt to ulike valuetypes? En for saksnummer fra fagsystem og en for saksnummer for fager. Ellers er jeg redd det kommer til gå i surr i fremtiden

Reply:
Nå har vi renamet metoder og variabler for å tydeliggjøre skillet. Hva synes du om det? Går an å lage egne typer også.